src/components/MovieCard.vue

```diff
@@ -48,37 +48,27 @@
       display: grid;
       grid-template-columns: 1fr;
       grid-template-rows: repeat(1, 1fr);
-      // grid-template-area: bottom;
       transition: all 0.2s ease-in-out;
       z-index: 0;
-      margin-bottom: 25px;
 
         img {
           height: 250px;  
           margin: 20px; 
           border-radius: 2px;
           border: solid .1px #d9d9d956; 
           box-shadow: 0 0 .75em #808080;
-          // grid-area: ;
         }
 
         &:hover {
-          @include transition;
           border: solid 3px $darkestRed;
         }
```

Comment:
Yeah, I think the hover state stuff was probably not the right approach given that it would complicate mobile.